core/backend/src/BriefcaseManager.ts

```diff
@@ -87,7 +85,7 @@ export class BriefcaseManager {
 
   /** @internal */
   public static getChangeSetFolderNameFromId(changeSetId: string): string {
-    return changeSetId || this._firstChangeSetDir;
+    return changeSetId || "first";

```

Comment:
What is this function for? I found no callers in imodeljs repo.

Reply:
I don't know what it was there for, I'll remove it.

Reply:
gone